lib/manager/components/ProjectFeedListToolbar.js

```diff
@@ -288,6 +293,57 @@ export default class ProjectFeedListToolbar extends PureComponent<Props> {
     return options
   }
 
+  _renderLabelFilter = () => {
+    const {
+      project,
+      setVisibilityLabel,
+      setVisibilityLabelMode,
+      filter
+    } = this.props
+    // This variable must be set directly from this.props both
+    // to allow the override and for flow
+    const labels = this.props.filter.labels || []
+    const { labelsFilterMode } = filter
+
+    function handleClick (labelId: string) {
+      const index = labels.indexOf(labelId)
+      index >= 0 ? labels.splice(index, 1) : labels.push(labelId)
+      setVisibilityLabel(labels)
+    }
```

Comment:
This one can also be a class method handler. Let's make them this way to be consistent and not be creating a new function on every render.

Reply:
Agreed. Extracted in a1bc3e9